dotCMS/src/main/java/com/dotmarketing/factories/DBTreeTransformer.java

```diff
@@ -0,0 +1,52 @@
+package com.dotmarketing.factories;
+
+import com.dotmarketing.beans.Tree;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Lists;
+
+
+public class DBTreeTransformer{
+    
+    private final List<Tree> trees;
+    
+    public DBTreeTransformer (List<Map<String, Object>>list ){
+        this.trees =  list
+        .stream()
+        .map(row -> toTree(row))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'row' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)